README.md

```diff
@@ -1,15 +1,105 @@
+### Overview
+This application serves as a reference implementation for using the [GraphQL](https://graphql.org/learn/) API of the
+[Trinity Cyber Customer Portal](https://portal.trinitycyber.com) to pull events and save them to a variety of sources,
+including the local filesystem and S3.
+
+The API itself follows [Relay GraphQL Cursor Connections Specification](https://relay.dev/graphql/connections.htm) for
+pagination. There is an [embedded GraphQL Playground](https://portal.trinitycyber.com/graphql-playground) embedded in
+our customer portal to try out some of these queries.
+
+This API works great for both ad-hoc queries as well as continual synchronization to another database or tool. This
+script is an example of how to query the API, save off the "end cursor" for each page of data processed, and emit
+events to be processed or submitted to another system.
+
 ### Requirements
-This is a Python 3 script; it was developd on CentOS 7 with Python 3.6 but should be broadly applicable to most Python 3 variants on most \*NIX based operating systems. The only additional library requried is `requests`.
+This is a Python 3 script; it was developed on CentOS 7 with Python 3.6 but should be broadly applicable to most Python
+3  variants on most UNIX based operating systems.  Library dependencies are listed in requirements.txt and can be
+installed using "pip install -r requirements.txt"
 
-### Overview
-The [Trinity Cyber Customer Portal](https://portal.trinitycyber.com) has a [GraphQL](https://graphql.org/learn/) API which uses the [Relay GraphQL Cursor Connections Specification](https://relay.dev/graphql/connections.htm) for pagination. There is an [embedded GraphQL Playground](https://portal.trinitycyber.com/graphql-playground) embedded in our customer portal to try out some of these queries.
+### Running as a Service
+If installing as a service using the RPM, the script and config files will be placed in the 
+/opt/trinity/tc-portal-sync/ directory.  Make sure any configuration files needed by the service
+are accessible to the "trinitycyber" user.
```

Comment:
I'm fine with either.   I kept the user from the S3 sync script (since all the RPM stuff was there).   One benefit for "trinitycyber" might be having a common user if we expand our scripts beyond sync -- not sure what the use case would be for that.

Reply:
That's what I was worried about - if we expand then the customer loses control over service user permissions.